tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -456,21 +456,15 @@ public function testCreateContributionWithPaymentInstrument() {
   public function testGetContributionByPaymentInstrument() {
     $params = $this->_params + array('payment_instrument' => 'EFT');
     $params2 = $this->_params + array('payment_instrument' => 'Cash');
-    $this->callAPISuccess('contribution', 'create', $params);
-    $this->callAPISuccess('contribution', 'create', $params2);
-    $contribution = $this->callAPISuccess('contribution', 'get', array(
-        'sequential' => 1,
-        'contribution_payment_instrument_id' => 'Cash'
```

Comment:
'contribution_payment_instrument_id' needs to continue to work as part of our test contract - in this case I would think it still would as it is the unique name for the field - but we shouldn't 'tidy the test for it' such that the existing contract is no longer tested